pylhc/constants/calibration.py

```diff
@@ -0,0 +1,133 @@
+"""
+Constants: BPM Calibration
+--------------------------
+
+Specific constants related to the BPM calibration script in ``PyLHC``.
+"""
+
+# IPS to be used to compute the calibraton factors
+IPS = [1, 4, 5]
+
+# Constants for TFS files
+LABELS = ['S',
+          'CALIBRATION',
+          'ERROR_CALIBRATION',
+          'CALIBRATION_FIT',
+          'ERROR_CALIBRATION_FIT']
+TFS_INDEX = 'NAME'
+D = 'D'
+ND = 'ND'
```

Comment:
since omc3 is also imported one could replace this with `from omc3.optics_measurements.constants import DISPERSION` ... but dispersion and normalized dispersion is not in the constants file bc it's hardcoded in `dispersion.py` so this is ok .... but should be done in omc3 and then ported to here